jenkins/Deploy.groovy.j2

```diff
@@ -47,7 +47,7 @@ def deploy_docs() {
       git status
       git checkout -B $DOCS_DEPLOY_BRANCH
 
-      rm -rf docs
+      git ls-tree HEAD docs/ | awk '{print $4}' | grep -v 'docs/v0.8.0' | xargs rm -rf
```

Comment:
i think you can like --name-only or something rather than awk. wanna do that? also...the `grep -v` doesn't seem especially scalable particularly as 0.9.0 release is getting to be imminent. i feel like we should just move the `main` docs to a subdir...thoughts?

Reply:
I swapped this so it should work with any folder that matches `docs/v\d` so we should be ok for the next release without changing this. We could move to `main` but we'd also need a redirect rule from the top level to not break existing doc URLs that are out there